README.md

```diff
@@ -172,20 +198,20 @@ public class JavaFuzzTest {
 }
 ```
 
-The corresponding build target looks very much like a regular `java_binary`:
+You should now define the corresponding target in the `BUILD` file, which looks very much like a regular `java_binary`:
 
 ```python
 load("@rules_fuzzing//fuzzing:java_defs.bzl", "java_fuzz_test")
 
 java_fuzz_test(
     name = "JavaFuzzTest",
-    srcs = ["JavaFuzzTest.java"],
+    srcs = ["com/example/JavaFuzzTest.java"],
     # target_class is not needed if using the Maven directory layout.
     target_class = "com.example.JavaFuzzTest",
```

Comment:
With the path change above, this allows to drop `target_class`:
```suggestion
    srcs = ["src/com/example/JavaFuzzTest.java"],
    # target_class is not needed if using the Maven directory layout.
    # target_class = "com.example.JavaFuzzTest",
```

Reply:
Done.